tests/shell/shell_recorder/set_validation.esr

```diff
@@ -0,0 +1,42 @@
+Mountpoint: user:/tests/set
+
+RET: 0
+< kdb meta-set spec:/tests/set mountpoint /tmp/spec.ni
+
+RET: 0
+< kdb meta-set spec:/tests/set/some/testKey1 type unsigned_short
+
+RET: 0
+< kdb spec-mount /tests/set
+
+STDOUT: unsigned_short
+RET: 0
+< kdb meta-get spec:/tests/set/some/testKey1 type
+
+STDERR: ^.*Validation Semantic: The type 'unsigned_short' failed to match for 'user:/tests/set/some/testKey1'.*$
+RET: 5
+< kdb set $Mountpoint/some/testKey1 Hello
+
+STDERR: ^.*Did not find key 'user:/tests/set/some/testKey1'.*$
+RET: 11
+< kdb get $Mountpoint/some/testKey1
+
+STDERR: ^.*Validation Semantic: The type 'unsigned_short' failed to match for 'user:/tests/set/some/testKey1'.*$
+RET: 5
+< kdb set $Mountpoint/some/testKey1 -- -5
+
+STDERR: ^.*Did not find key 'user:/tests/set/some/testKey1'.*$
+RET: 11
+< kdb get $Mountpoint/some/testKey1
+
+RET: 0
+< kdb set $Mountpoint/some/testKey1 5
+
+RET: 0
```

Comment:
Please also add a test case for `--force`.

Reply:
done